src/pages/form/[id].tsx

```diff
@@ -1,21 +1,32 @@
 import { api } from "../../utils/api";
 import JsonForm from "../../components/forms/JsonForm";
-import { useMemo } from "react";
+import { useState, useMemo } from "react";
 import { useRouter } from "next/router";
 import { type JSONObject } from "superjson/dist/types";
 import DefaultLoadingPage from "../../components/loading/loading";
 import ErrorPage from "../../components/error/error";
+import { Card } from "@mui/material";
 import { Container, Typography } from "@mui/material";
+import { useSession } from "next-auth/react";
 
 /**
  * Form page that gets the collection by id and renders the form based
  * on the schemas and uiSchemas from the collection
  */
 const Form = () => {
+  const [submitted, setSubmitted] = useState(false);
+  const [countdown, setCountdown] = useState(5);
   const router = useRouter();
   const { data, error, isLoading } = api.collection.getCollectionById.useQuery({
     id: router.query.id as string,
   });
+  const {
+    data: submssionData,
+    error: submissionError,
+    isLoading: submissionLoading,
+  } = api.collection.getSubmissions.useQuery({
```

Comment:
A user shouldn't be able to call this query, only admins. Can we instead maybe do getSubmissionByCollectionId which a user can run if and only if the submission was submitted by them? If it returns null then the user hasn't filled out the form